auth/store_test.go

```diff
@@ -585,7 +608,11 @@ func TestRecoverFromSnapshot(t *testing.T) {
 
 	as.Close()
 
+<<<<<<< HEAD
```

Comment:
@cfc4n I noticed this while working on another PR. Trying to fix this under https://github.com/etcd-io/etcd/pull/12087

Reply:
Thanks you. 😄